homeassistant/components/iqvia/config_flow.py

```diff
@@ -54,11 +54,10 @@ def __init__(self):
             return await self._show_form({CONF_ZIP_CODE: 'identifier_exists'})
 
         websession = aiohttp_client.async_get_clientsession(self.hass)
-        client = Client(user_input[CONF_ZIP_CODE], websession)
 
         try:
-            await client.allergens.current()
-        except IQVIAError:
+            Client(user_input[CONF_ZIP_CODE], websession)
```

Comment:
Will this make a request? Or is the zip code validated with static info?

Reply:
It's validated immediately when the `Client` is created.